packages/common/src/ERC4337Utils.ts

```diff
@@ -1,14 +1,22 @@
 import { arrayify, defaultAbiCoder, keccak256 } from 'ethers/lib/utils'
+import { UserOperationStruct } from '@account-abstraction/contracts'
+import { abi as entryPointAbi } from '@account-abstraction/contracts/artifacts/IEntryPoint.json'
 
-import { UserOperation } from './UserOperation'
+// UserOperation is the first parameter of simulateValidation
+const UserOpType = entryPointAbi.find(entry => entry.name === 'simulateValidation')?.inputs[0]
```

Comment:
It is not a reliable way to find a structure description, maybe it is better to just copy it and commit the file?